src/strategy-heat-capacitor-functions.js

```diff
@@ -0,0 +1,181 @@
+const { DateTime } = require("luxon");
+//const decreasing_end_prices = require("../test/data/tibber-decreasing-24h.json");
+
+function calculate_opportunities(prices, pattern, amount){
+    //creating a price vector with minute granularity
+    var temp_price = Array(prices.length*60).fill(0)
+    var end_index=[];
+    for(i=0;i<prices.length;i++){
```

Comment:
Always use `let` when defining for loop variables. `let i = 0`

Reply:
Done. At least I think so :-)